Project_0.Altis/AI/Garrison/ActionGarrisonSurrender.sqf

```diff
@@ -0,0 +1,36 @@
+#include "common.hpp"
+
+/*
+Class: ActionGarrison.ActionGarrisonSurrender
+*/
+
+CLASS("ActionGarrisonSurrender", "ActionGarrison")
+
+	METHOD("activate") {
+		params [["_thisObject", "", [""]]];
+
+		private _AI = T_GETV("AI");
+		private _gar = T_GETV("gar");
+
+		// Send surrender goal to group for more relevance
+		private _groups = CALLM0(_gar, "getGroups");
+		{
+			private _groupAI = CALLM0(_x, "getAI");
+			CALLM4(_groupAI, "addExternalGoal", "GoalGroupSurrend", 0, [], gAICommanderEast);
```

Comment:
"GoalGroupSurrend", while Group goal is named properly "...surrender". I think you should make sure that it works so that we don't have to return to it when we need to use it.

Reply:
good catch @Sparker95 thx !